ugandaemr_mobile/uninstall-ugandaemr-mobile.md

```diff
@@ -0,0 +1,7 @@
+# Uninstalling UgandaEMR Mobile
```

Comment:
@solemabrothers Move this to the bottom of the installation screen

Reply:
Done